portfolio/src/main/webapp/script.js

```diff
@@ -19,10 +19,25 @@ function addRandomGreeting() {
   const greetings =
       ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
+
   // Pick a random greeting.
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
+
+
+
 }
+
+function showGreeting() {
+
+//   print("Is showGreeting running?");
+//   <p>Does this work inside a JavaScript file</p>
+  const myGreeting = "Something I do to study Japanese is to go to Twitter and read random Japanese tweets"
+  
+  const myGreetingContainer = document.getElementById('my-greeting-container');
+  myGreetingContainer.innerText = myGreeting;
+
+}
```

Comment:
Add trailing newline.

Reply:
This is what the red circle symbol here means. That you don't have a literal `\n` character at the end of the `script.js` file.